src/components/button/button.md.scss

```diff
@@ -484,6 +484,12 @@ $button-md-strong-font-weight:                  bold !default;
   transition-timing-function: ease-in-out;
 
   pointer-events: none;
+
+  @include multi-dir() {
+    top: 0;
+
```

Comment:
sure but css lint gave me error because I hadn't put empty line in between ... I think it was because I had order problems? I'll commit the new changes asap